### ISSUE TITLE
[Request New Content] What Is Multi-Tenanted Software

### ISSUE THREAD

**Issue by endymion1818** [webiny/write-with-webiny#94]
### Topic

We'd like to highlight the benefits of Webiny's Multi-tenant system, to cover:

- What is Multi-Tenancy
- Why You Might Need a Multi-Tenant System
- Drawbacks of not using a Multi-Tenant System
- - a. Large organisation
- - b. Large number of products
- - c. Large content team
- Which other Open Source CMS have Multi-Tenancy
- Webiny's Approach to Multi-Tenancy

#### References
https://www.webiny.com/docs/overview/features/multi-tenancy

#### Community Vote
👍 If you like this topic, please upvote it!

**Comment by Saintdavidking** [comment]
Hi @endymion1818 is this topic still availabe?

**Comment by endymion1818** [comment]
@Saintdavidking Yes it's available, I'll send you an agreement for this article shortly.

**Comment by Saintdavidking** [comment]
@endymion1818 Alright, thanks

**Comment by Saintdavidking** [comment]
@endymion1818 I haven't seen the agreement

**Comment by endymion1818** [comment]
Sorry @Saintdavidking, I've been off because the family have had COVID. All better now thankfully, I'll send you something soon!

**Comment by Saintdavidking** [comment]
@endymion1818, so sorry for the inconvenience, Thanks goodness all is better now. I think you can take your time to make sure everything is fine. so sorry

**Comment by Saintdavidking** [comment]
Hi @endymion1818, can I proceed with this article?

**Comment by endymion1818** [comment]
Hi @Saintdavidking, I've just posted a note about this type of article in the Slack channel for writers, please see there for more detail as to why we're holding off on these ones for now. Thanks for your interest in writing this one.

**Comment by danielonugha0** [comment]
Hello @endymion1818 I would love to write on this topic

**Comment by Kalderon-Sheikhman** [comment]
Hello @endymion1818 , please is this article still availble?
I would like to write on it
